Yoast/javascript
# Create ImageSelect component

## Summary


This PR can be summarized in the following changelog entry:

* Adds an ImageSelect component with an image preview or url preview

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Run `yarn start` in `apps/components`
* Go to image select
* When clicked on the image preview or "select image" button, a placeholder image should appear
* Make sure you can also remove the image by clicking "remove image" button
* In ImageSelectWrapper.js change `hasPreview` to `false` and you should now see a disabled input field
* With an image selected, this input field should show the url of the image

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes P1-187
